docs/getting-started.md

```diff
@@ -74,7 +74,75 @@ To create a new headless WordPress app using SnapWP, follow these steps:
 
 ### Manual Installation
 
-@todo
+1. Install npm packages in your Next App
+
+```bash
+npm i @snapwp/blocks @snapwp/core @snapwp/next @snapwp/query
```

Comment:
In user-facing docs we should avoid aliases.

Also, do we want a save flag so these are stored in their package.json?

```suggestion
npm install @snapwp/blocks @snapwp/core @snapwp/next @snapwp/query
```